core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -108,47 +100,18 @@ public MasterWorkerInfo(long id, WorkerNetAddress address) {
    * @param blocks set of block ids on this worker
    * @return A Set of blocks removed (or lost) from this worker
    */
+  // TODO(jiacheng): Do I need to write GuardedBy for member methods like this?
```

Comment:
I have 2 questions:
1. Do I need to write GuardedBy on this kind of member methods, given that I declared this object needs locking logic externally? I guess the most informative approach is yes.
2. AFAIK `GuardedBy` doesn't allow specifying multiple locks, what should I do?

Reply:
@apc999 @ggezer PTAL I have some small questions on the coding style, thanks!

Reply:
1- methods doesn't make sense to me for this annotation.
2- I think every individual member should be protected by a single lock only. So unless applying on a method, we shouldn't need to specify multiple locks.